contrib/devtools/check-rpc-mappings.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/python3
+'''
```

Comment:
nit: One-line docstrings should be on one line.

Reply:
Meh, I personally prefer them on multiple lines, makes it easier to extend them later.

Reply:
Thanks for changing it anyway! (for context, I picked up the preference for one-line docstrings from https://www.python.org/dev/peps/pep-0257/#id16)